apps/src/tutorialExplorer/filterHeader.jsx

```diff
@@ -13,30 +13,46 @@ const styles = {
     paddingRight: 7
   },
   bar: {
-    backgroundColor: "#eee",
-    height: 42
+    backgroundColor: "rgb(0, 178, 192)",
+    color: "white",
+    minHeight: 44,
+    overflow: "hidden"
+  },
+  select: {
+    backgroundColor: "rgb(101, 205, 214)",
+    color: "white",
+    borderColor: "white",
+    height: 34
   },
   filterBy: {
     float: "left",
-    lineHeight: "42px",
+    lineHeight: "44px",
+    marginLeft: 10
+  },
+  left: {
+    float: "left",
+    lineHeight: "44px",
     marginLeft: 10
   },
   right: {
     float: "right",
-    lineHeight: "42px",
+    lineHeight: "44px",
     marginRight: 10
   }
 };
 
 const FilterHeader = React.createClass({
   propTypes: {
+    onUserInput: React.PropTypes.func.isRequired,
+    sortBy: React.PropTypes.string.isRequired,
```

Comment:
This could probably be:
`sortBy: React.PropTypes.oneOf(Object.keys(SortBy)).isRequired`